Makefile.am

```diff
@@ -17,8 +17,8 @@ doxygen_doc_files = $(noinst_HEADERS)
 perftest_dir = $(pkgdatadir)/perftest
 doc_dir = $(pkgdatadir)/doc
 
-dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
-dist_doc__DATA = README
+dist_perftest__DATA = contrib/ucx_perftest_config/msg_pow2 contrib/ucx_perftest_config/README.md contrib/ucx_perftest_config/test_types contrib/ucx_perftest_config/transports
```

Comment:
But I don't want to have duplicated files ? The latest commit fixes the issue that everything supposed to work